spec/spec_helper.rb

```diff
@@ -2,7 +2,6 @@
 #   https://github.com/theforeman/foreman-installer-modulesync
 
 require 'puppetlabs_spec_helper/module_spec_helper'
-require 'lib/module_spec_helper'
```

Comment:
I was looking at our modules and I wonder if we should include that helper in modulesync.